File: main (1).cpp
/******************************************************************************

                              Online C++ Compiler.
               Code, Compile, Run and Debug C++ program online.
Write your code in this editor and press "Run" button to compile and execute it.

*******************************************************************************/

#include <iostream>
#include <bits/stdc++.h>

using namespace std;
float slope(float x1, float y1, float x2 ,float y2){
    if(x2 - x1 != 0){
        return (y2 - y1) / (x2 -x1);
    }
    return INT_MAX;
}
int main()
{
    float x1 = 4, y1 = 2;
    float x2 = 2, y2 = 5;
    cout<<slope(x1, y1, x2, y2);
    return 0;
}

File: main (1).py
'''

Welcome to GDB Online.
GDB online is an online compiler and debugger tool for C, C++, Python, Java, PHP, Ruby, Perl,
C#, OCaml, VB, Swift, Pascal, Fortran, Haskell, Objective-C, Assembly, HTML, CSS, JS, SQLite, Prolog.
Code, Compile, Run and Debug online from anywhere in world.

'''

s = int(input("no of enetr ele"))
lis = []
pos = int(input("position"))
ele = int(input("element"))
for i in range(s):
    lis.append(int(input()))
temp = lis[-1]

for i in range(len(lis)-1, pos, -1):
    lis[i] = lis[i-1]
lis[pos] = ele
lis.append(temp)
print(lis)
